be/src/common/stack_trace.cpp

```diff
@@ -439,7 +439,8 @@ static StackTraceCache& cacheInstance() {
 
 static std::mutex stacktrace_cache_mutex;
 
-std::string toStringCached(const StackTrace::FramePointers& pointers, size_t offset, size_t size) {
+std::string toStringCached(const StackTrace::FramePointers& pointers, size_t offset, size_t size,
```

Comment:
do not modify these very low API, it is very likely to import some bugs.

Reply:
i am 99.99% sure it's fine